LibTest/js_interop/JSAnyOperatorExtension/strictEquals_t01.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion JSBoolean strictEquals( JSAny? any )
+/// The result of `this === any` in JavaScript.
+///
+/// @description Check that `strictEquals` returns result of `this === any` in
+/// JavaScript.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+import '../../../Utils/expect.dart';
+import '../js_utils.dart';
+
+testStrictEquals(JSAny? underTest) {
+  eval(r'''
+    var eqNum = underTest === 2; 
+    var eqString = underTest === "text";
+    var eqNull = underTest === null;
+    var eqUndefined = underTest === undefined;
+    var eqNaN = underTest === (0 / 0);
+    var eqArray = underTest === [1, 2];
+    var eqTrue = underTest === true;
+  ''');
+
+  jsExpectEquals(globalContext["eqNum"], underTest.strictEquals(2.toJS));
+  jsExpectEquals(
+    globalContext["eqString"],
+    underTest.strictEquals("text".toJS),
+  );
+  jsExpectEquals(globalContext["eqNaN"], underTest.strictEquals((0 / 0).toJS));
+  jsExpectEquals(
+    globalContext["eqArray"],
+    underTest.strictEquals([1, 2].jsify()),
+  );
+  jsExpectEquals(globalContext["eqTrue"], underTest.strictEquals(true.toJS));
+
+  if (isJS) {
```

Comment:
I see you've come across our discrepancy on how we have to convert `undefined` to Dart `null` in dart2wasm :D

Reply:
Yes :)